### PR TITLE
{lib}[foss/2019b] tensorflow-probability v0.9.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by dithwick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('TensorFlow', '2.1.0', versionsuffix),
+]
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'sanity_pip_check': True,
+}
```
Comment (boegel) [inline easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb:30]: @dithwick We usually set `sanity_pip_check` "top-level" below (near `sanity_check_commands`):

```python
exts_default_options = {'source_urls': [PYPI_SOURCE]}

use_pip = True

exts_list = [
    ...
]

sanity_check_commands = ["python -c 'import tensorflow; import tensorflow_probability'"]

sanity_pip_check = True
```

File: easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    (name, version, {
+     'modulename': 'tensorflow_probability',
+     'source_tmpl': 'tensorflow_probability-%(version)s-py2.py3-none-any.whl',
+     'checksums': ['66928dcbbc12987e574232a576bd9643ba7ca6b2a837f2182417286ac955d50f'],
+     'unpack_sources': False,
+     }),
+]
+
+sanity_check_commands = [
+    """python -c 'import tensorflow; import tensorflow_probability' """,
```
Comment (boegel) [inline easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb:46]: this can be a one-liner, see above

Comment (dithwick) [reply easybuild/easyconfigs/t/tensorflow-probability/tensorflow-probability-0.9.0-foss-2019b-Python-3.7.4.eb]: Thanks, just making and committing the changes.

**Comment by dithwick** [comment]
I've only just noticed, some of the checks didn't complete, it's showing "Expected - Waiting for status to be reported"!

**Comment by boegel** [comment]
close/re-open to re-trigger CI tests

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/ebd1693619278e875f83350aca05f790 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @dithwick!

**Pull request merged**